hackathon_site/dashboard/frontend/src/components/general/OrderTables/OrderTables.tsx

```diff
@@ -21,6 +23,19 @@ import hardwareImagePlaceholder from "assets/images/placeholders/no-hardware-ima
 import { useSelector } from "react-redux";
 import { hardwareSelectors } from "slices/hardware/hardwareSlice";
 
+function formatDateTime(dateTimeString: string): string {
```

Comment:
Can you move this function to helpers.ts? Also can you use arrow notation for this function like this:
`export const formatDateTime = (dateTimeString: string): string => {`

Reply:
Apologies, I did have another question. Do you know why the tags for checked out items positioned to the left? Is it possible if you can reallign so its the same for all tables
![image](https://github.com/ieeeuoft/hackathon-template/assets/43456930/1b87cd8c-0c4d-451f-9cfe-7a937c4b83b4)